demoextendgrid/demoextendgrid.php

```diff
@@ -52,6 +52,23 @@ public function install()
         return $installer->install($this);
     }
 
+    public function hookActionAdminControllerSetMedia(array $params)
+    {
+        // check if it is orders controller
+        if ($this->context->controller->controller_name !== 'AdminOrders') {
+            return;
+        }
+        $action = Tools::getValue('action');
+
+        // check if it is orders index page (we want to skip if it is `order create` or `order view` page)
```

Comment:
Does this work for both 1.7.6 (legacy AdminOrder page) and 1.7.7 (Order page migrated to Symfony)?

Reply:
My bad it's 1.7.7+